src/main/java/pl/javny/expensesrepository/data/ReckonLogic.java

```diff
@@ -0,0 +1,39 @@
+package pl.javny.expensesrepository.data;
+
+import java.util.Scanner;
+
+public class ReckonLogic {
+
+    private final String save = "1";
+    private final String update = "2";
+    private final String delete = "3";
+    private final String incomes = "4";
+    private final String outcomes = "5";
+
+    public void runAppLogic(Scanner scanner) {
+        while (true) {
+
+            printOptions();
+            String userInput = scanner.nextLine();
+
+            switch (userInput) {
+                case save -> ReckonSave.main(new String[0]);
+                case update -> ReckonUpdate.main(new String[0]);
+                case delete -> ReckonDeleteRow.main(new String[0]);
+                case incomes -> ReckonIncome.main(new String[0]);
+                case outcomes -> ReckonOutcome.main(new String[0]);
+                default -> System.out.println("Błędny wybór!");
```

Comment:
Jeżeli stworzyłeś sobie logikę z pętlą, to nie rozumiem, dlaczego metod od zapisu, czy usuwania nie zebrałeś w jednej klasie, tylko w osobnych klasach z metodami main.
Te osobne klasy byłyby ok, gdybyś nie miał tutaj pętli i każdą klasę chciałbyś uruchamiać osobno, a tak to trochę się ze sobą kłóci